doc/ref/language.xml

```diff
@@ -539,9 +539,34 @@ Certain special facilities are provided for manipulating global
 variables which are not available for other types of variable (such as 
 local variables or function arguments).
 <P/>
-First, such variables may be marked <E>read-only</E>. In which case
+First, such variables may be marked <E>read-only</E> using
+<Ref Func="MakeReadOnlyGlobal"/>. In which case
 attempts to change them will fail. Most of the global variables
-defined in the &GAP; library are so marked.
+defined in the &GAP; library are so marked. <E>read-only</E> variables
+can be made read-write again by calling <Ref Func="MakeReadWriteGlobal"/>.
+GAP also features <E>constant</E> variables,
```

Comment:
Why the weird wrapping here?

Reply:
Also, GAPDoc recommends to use entity `&GAP;` for GAP